compiler/damlc/tests/src/DA/Test/ShakeIdeClient.hs

```diff
@@ -584,7 +584,8 @@ goToDefinitionTests mbScenarioService = Tasty.testGroup "Go to definition tests"
             expectGoToDefinition (foo,2,[2..4]) Missing             -- " : "
             expectGoToDefinition (foo,2,[9])    Missing             -- "\n"
             expectGoToDefinition (foo,2,[10])   Missing             -- (out of range)
-            expectGoToDefinition (foo,3,[0..5]) (At (foo,3,0))      -- "foo = "
+            expectGoToDefinition (foo,3,[0..2]) (At (foo,3,0))      -- "foo"
```

Comment:
This simply undoes the changes that I made in c65f9181459cadc3da79687571788bdbd74c4484 which were required by the regression in https://github.com/digital-asset/ghcide/issues/315. Should have taken a closer look at this back then